File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/C/PRAK401-2310817310002-JovanGilbertNatamasindah.c
#include <stdio.h>

void main() {
    int  ANGKA;
    char SIMBOL;

    printf("Masukkan Angka dan Simbol : ");
    scanf("%d %c",&ANGKA, &SIMBOL);

    for(int BILANGAN = 1 ; BILANGAN <= 50 ; BILANGAN++ ) {

        if (BILANGAN % ANGKA == 0) {
            printf("%c ", SIMBOL);
            continue;
        }
        
        printf("%d ", BILANGAN);
    }
}

File: semester I/Praktikum Pemrograman I/Modul4_JovanGilbertNatamasindah_2310817310002/Python/PRAK401-2310817310002-JovanGilbertNatamasindah.py
ANGKA, SIMBOL = input("Masukkan Angka dan Simbol : ").split(" ")

for (BILANGAN) in range(1,51) :
    
    if (int(BILANGAN )% int(ANGKA) == 0) :
        print(SIMBOL, end=" ")
        continue

    print(int(BILANGAN), end=" ")



# penjelasan mengenai penggunaan end="" : https://www.freecodecamp.org/news/python-for-loop-example-how-to-write-loops-in-python/